README.md

```diff
@@ -95,6 +95,16 @@ Each Minim type provides the following attributes:
 
 Each Minim type provides the following the methods.
 
+#### elementName
+
+When called, this returns the name of the element. When subclassing a Refract primitive, create this method on the new class to set a new element name.
```

Comment:
This should be addressed in [this](https://github.com/refractproject/minim/commit/31751160d1755e154b92b188afcd311a31889c7f) commit.